tmk_core/protocol/vusb/main.c

```diff
@@ -94,6 +99,10 @@ int main(void) {
             // To prevent failing to configure NOT scan keyboard during configuration
             if (usbConfiguration && usbInterruptIsReady()) {
                 keyboard_task();
+
+#if defined(RGBLIGHT_ANIMATIONS) & defined(RGBLIGHT_ENABLE)
```

Comment:
Shouldn't this be: 
```suggestion
#if defined(RGBLIGHT_ANIMATIONS) && defined(RGBLIGHT_ENABLE)
```

Reply:
hmmm copy paste from <https://github.com/qmk/qmk_firmware/blob/master/tmk_core/protocol/lufa/lufa.c#L1004> but its worth fixing anyway